### ISSUE TITLE
Update email settings

### ISSUE THREAD

**Issue by meomancer** [kartoza/IGRAC-GGIS#360]
Currently our email settings are:
https://github.com/kartoza/IGRAC-GGIS/blob/develop/django_project/core/settings/prod.py#L13

```
# See fig.yml file for postfix container definition#
EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
# Host for sending e-mail.
EMAIL_HOST = 'smtp'
# Port for sending e-mail.
EMAIL_PORT = 25
# SMTP authentication information for EMAIL_HOST.
# See fig.yml for where these are defined
EMAIL_HOST_USER = 'noreply@kartoza.com'
EMAIL_HOST_PASSWORD = 'docker'
EMAIL_USE_TLS = False
EMAIL_SUBJECT_PREFIX = '[IGRAC-GGIS]'

SERVER_EMAIL = os.environ.get('ADMIN_EMAILS', 'admin@kartoza.com')
```

it makes igrac email turns as spam
and also we need to change every settings that coming from kartoza

could you check what should we change in here @lucernae 
and also @ArnaudIGRAC , maybe your input in here? example : EMAIL_HOST_USER is the email of igrac system

**Comment by meomancer** [comment]
it's mentioned in:
https://github.com/kartoza/IGRAC-GGIS/issues/317#issuecomment-760795781

**Comment by meomancer** [comment]
we need to direct this question to stefan

**Comment by meomancer** [comment]
we keep current smtp setup for now and revisit it in future

**Comment by timlinux** [comment]
See email thread between Arnaud, Stefan and Kartoza, we agreed to park this for now, acknowledging that for many users, notification emails will go to spam.

**Comment by lucernae** [comment]
Hi, some update.

@meomancer asked me to setup internal SMTP service using our normal `catatnight/postfix` docker image setup.
Credentials used are exactly the same as specified [here](https://github.com/kartoza/IGRAC-GGIS/issues/360#issue-799996146), except for the password (I generate a more secure password).

The email indeed came if you check the spam folder.

There are some question from me though:

Maybe it's better to change some settings:
- `SERVER_EMAIL`: should go to IGRAC's email address that maintains the server. Because server's notification will go here. If we set it to `admin@kartoza.com` it will go to our admin email address.
- `DEFAULT_FROM_EMAIL`: should go to IGRAC`s reachable email address representatives. This is the address that will be listed to reply back for other user that receives notification. If user received email from GGIS, then if I replied the email, it will go to this address.

My recommendation would be:
- `SERVER_EMAIL`: should be set to IGRAC's GGIS IT admin (for logging purposes). If not, let's set it to a blackhole email address like `noreply@kartoza.com`
- `DEFAULT_FROM_EMAIL`: should be set to IGRAC's GGIS human admin, in case user want to communicate. Otherwise, just set it to a blackhole email address like `noreply@kartoza.com`

I will set both to `noreply@kartoza.com` for now to reduce noise.
